core/controllers/admin.py

```diff
@@ -656,6 +658,453 @@ def _generate_dummy_explorations(
+            'content_id': 'hint_id',
+            'language_code': 'ak',
+            'content_html': '<p>This is a hint.</p>',
+            'state_name': state_name,
+            'translation_html': '<p>Translation for content 2</p>',
+            'data_format': 'html'
+        }
+        change_dict3 = {
+            'cmd': 'add_written_translation',
+            'content_id': 'feedback_id',
+            'language_code': 'ak',
+            'content_html': '<p>Dummy Feedback</p>',
+            'state_name': state_name,
+            'translation_html': '<p>Translation for content 3</p>',
+            'data_format': 'html'
+        }
+        change_dict4 = {
+            'cmd': 'add_written_translation',
+            'content_id': 'solution',
+            'language_code': 'ak',
+            'content_html': '<p>This is a solution.</p>',
+            'state_name': state_name,
+            'translation_html': '<p>Translation for content 4</p>',
+            'data_format': 'html'
+        }
+        suggestion_services.create_suggestion(
+        feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+        feconf.ENTITY_TYPE_EXPLORATION,
+        exp_id, 3, self.user_id, change_dict1, 'description')
+
+        suggestion_services.create_suggestion(
+        feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+        feconf.ENTITY_TYPE_EXPLORATION,
+        exp_id, 3, self.user_id, change_dict2, 'description')
+
+        suggestion_services.create_suggestion(
+        feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+        feconf.ENTITY_TYPE_EXPLORATION,
+        exp_id, 3, self.user_id, change_dict3, 'description')
+
+        suggestion_services.create_suggestion(
+        feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+        feconf.ENTITY_TYPE_EXPLORATION,
+        exp_id, 3, self.user_id, change_dict4, 'description')
+
+    def _generate_sample_chapters(
+        self, num_sample_ops,
+        num_sample_interactions_per_opportunity, num_chapters):
+        """Generates sample chapters and creates a list of story_node_dicts.
+
+        Args:
+            num_sample_ops: int. Number of sample opportunities that have
+                been generated.
+            num_sample_interactions_per_opportunity: int. Number of sample
+                interactions that have been generated per opportunity.
+            num_chapters: int. Current number of chapters in
+                story.
+
+        Returns:
+            story_node_dicts. List. A list of dicts that represent story nodes.
+        """
+        story_node_dicts = []
+
+        possible_titles = ['Hulk Neuroscience', 'Quantum Starks',
+                            'Wonder Anatomy',
+                            'Elvish, language',
+                            'The Science of Superheroes']
+        exploration_ids_to_publish = []
+        for opportunity_num in range(num_sample_ops):
+            title = random.choice(possible_titles) + ' ' + str(
+                opportunity_num + num_chapters)
```

Comment:
Prefer using string interpolation instead of concatenation here (easier to read).